templates/web/bromley/report/display.html

```diff
@@ -24,144 +24,147 @@
 [% INCLUDE 'report/display_tools.html' %]
 [% INCLUDE 'report/updates.html' %]
 
-<div id="update_form">
-    <h2>[% loc( 'Provide an update') %]</h2>
-
-    [% INCLUDE 'errors.html' %]
-
-    <form method="post" action="[% c.uri_for( '/report/update' ) %]" name="updateForm" class="validate"[% IF c.cobrand.allow_photo_upload %] enctype="multipart/form-data"[% END %]>
-        <input type="hidden" name="token" value="[% csrf_token %]">
-        <fieldset>
-            <input type="hidden" name="submit_update" value="1">
-            <input type="hidden" name="id" value="[% problem.id | html %]">
-
-        [% IF c.cobrand.allow_photo_upload %]
-            <input type="hidden" name="upload_fileid" value="[% upload_fileid %]">
-            <label for="form_photo">
-                <span data-singular="[% loc('Photo') %]" data-plural="[% loc('Photos') %]">[% loc('Photo') %]</span>
-            </label>
-
-              [% IF field_errors.photo %]
-                <p class='form-error'>[% field_errors.photo %]</p>
-              [% END %]
-
-            <div id="form_photos">
-              [% IF upload_fileid %]
-                <p>[% loc('You have already attached photos to this update.  Note that you can attach a maximum of 3 to this update (if you try to upload more, the oldest will be removed).') %]</p>
-                [% FOREACH id IN upload_fileid.split(',') %]
-                <img align="right" src="/photo/temp.[% id %]" alt="">
+[% IF problem.duplicate_of %]
+    [% INCLUDE 'report/duplicate-no-updates.html' %]
+[% ELSE %]
```

Comment:
Can we factor out this form like the base does to prevent all this whitespace change?